contracts/bid/src/map.rs

```diff
@@ -4,23 +4,20 @@
 //
 // Copyright (c) DUSK NETWORK. All rights reserved.
 
-use canonical::{Canon, Store};
+use canonical::CanonError;
 use canonical_derive::Canon;
 use core::ops::Deref;
 use dusk_bytes::Serializable;
-use dusk_kelvin_map::Map;
+use dusk_hamt::Hamt;
 use dusk_pki::PublicKey;
 
 #[derive(Default, Debug, Clone, Canon)]
-pub struct KeyToIdxMap<S: Store>(Map<[u8; 32], u64, S>);
+pub struct KeyToIdxMap(Hamt<[u8; 32], u64, ()>);
```

Comment:
It's okay to keep as is but you can also use the type alias `Map` (where the tuple is implicit so you only have two arguments)